package.json

```diff
@@ -8,7 +8,8 @@
     "build:ci": "npm run build && npm run build:cionly",
     "build:cionly": "tsc --build ci",
     "start": "next start",
-    "lint": "next lint",
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
```

Comment:
In -types I target just the folders I need, and I think you should too in here 👀

Reply:
I mean...isn't that what the config is for? Did it though